src/gui/touchscreengui.cpp

```diff
@@ -909,7 +919,8 @@ void TouchScreenGUI::translateEvent(const SEvent &event)
 
 		if (m_move_id != -1) {
 			if ((event.TouchInput.ID == m_move_id) &&
-				(!m_move_sent_as_mouse_event)) {
+				(!m_move_sent_as_mouse_event || m_use_crosshair ||
+				m_camera_mode == CAMERA_MODE_THIRD)) {
```

Comment:
These two lines need to be indented by one more tab to not align with the code within the if block

Reply:
Done.